libs/test-utils-vitest/package.json

```diff
@@ -18,7 +18,7 @@
     "lint:fix": "pnpm type-check && eslint . --fix",
     "pre-commit": "lint-staged",
     "test": "is-ci test:ci test:watch",
-    "test:ci": "vitest --coverage --reporter junit --outputFile reports/junit.xml",
+    "test:ci": "vitest --coverage",
     "test:coverage": "vitest run --coverage",
```

Comment:
I think these should be switched - we want run mode in CI and watch mode in local dev, right? (This is my mistake, I'm just noticing it now)

Reply:
Done, though if `CI=true` then `vitest` already doesn't use watch mode by default.

Reply:
Oh, that's good to know. In that case I don't care as much